hedgehog-extras.cabal

```diff
@@ -34,7 +34,7 @@ common network                      { build-depends: network
 common process                      { build-depends: process                                                      }
 common resourcet                    { build-depends: resourcet                                                    }
 common stm                          { build-depends: stm                                                          }
-common tar                          { build-depends: tar                                                          }
+common tar                          { build-depends: tar                              == 0.5.1.1                  }
```

Comment:
I believe we could use caret here if we simply don't want tar 0.6 i.e. `^>= 0.5`